pages/common/scrot.md

```diff
@@ -0,0 +1,15 @@
+# scrot
+
+> Capture screenshots using the command line.
+
+- Capture the whole screen:
+
+`scrot`
+
+- Select a window with a mouse then capture the window:
+
+`scrot -s`
+
+- Wait *sec* seconds before taking a screenshot and display a countdown:
```

Comment:
Perhaps it would work better giving a concrete example in this case. Say, 5 seconds?